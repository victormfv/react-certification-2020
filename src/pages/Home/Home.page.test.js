import React from 'react';
import renderer from 'react-test-renderer';
import HomePage from './Home.page';

describe('Home Page', () => {

    it('renders correctly', () => {
      const tree = renderer.create(<HomePage/>).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });