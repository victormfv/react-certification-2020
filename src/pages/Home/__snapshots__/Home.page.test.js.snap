// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page renders correctly 1`] = `
<section
  className="homepage"
>
  <h2>
    Welcome to the Challenge!
  </h2>
  <div
    className="sc-eCssSg iUvlzN"
  >
    
  </div>
</section>
`;
